addons/website_blog/views/website_blog_components.xml

```diff
@@ -21,9 +22,62 @@
                     </a>
                 </li>
             </ul>
+            <!-- Mobile -->
+            <t t-set="wblog_nav_offcanvas" t-value="len(blogs) > 4"/>
+            <t t-set="wblog_all_title_string">All Blogs</t>
+
+            <div t-if="len(blogs) > 1" t-attf-class="order-2 d-lg-none #{not wblog_nav_offcanvas and 'dropdown flex-grow-1 flex-sm-grow-0 min-w-0'}">
+                <a t-attf-class="btn btn-light #{not wblog_nav_offcanvas and 'dropdown-toggle d-flex align-items-center justify-content-between'}"
+                   type="button"
```

Comment:
An anchor tag shouldn't have a `type="button"` attribute, `type` it's used for MIME ([ref.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#type))